lib/src/sdk_api/models/conference.dart

```diff
@@ -210,11 +210,8 @@ class ConferenceJoinOptions {
   /// Allows joining a conference as a special participant called Mixer. For more information, see the [Recording Conferences](https://docs.dolby.io/communications-apis/docs/guides-recording-conferences) article.
   ConferenceMixingOptions? mixing;
 
-  /// Indicates whether a participant wants to receive mono sound. By default, participants receive stereo audio. This configuration is only applicable when using the Opus codec.
-  bool? preferRecvMono;
-
-  /// Indicates whether a participant wants to send mono sound to a conference. By default, when using the Opus codec, participants' audio is sent as stereo. This configuration is only applicable when using the Opus codec.
-  bool? preferSendMono;
```

Comment:
Why are we getting rid of these?

Reply:
The question is why? And also should they be used?

Reply:
React native copied Web SDK parameters and we copied it from react native. There is no equivalent parameters in native mobile SDKs.